summarization/stamp_page_picking/scoring_utils.py

```diff
@@ -102,11 +85,4 @@ def _compute_weighted_score(self, score_list):
     def _get_stamp_page_type(self, stamp_page_index):
```

Comment:
Could this method be changes to accept a stamp_page directly? Also, rename the method to `get_stamp_page_value_from_type()` (something that expresses the functionality more).

Reply:
updated!